style.css

```diff
@@ -0,0 +1,55 @@
+body {
+  background-image: url("../img/bg.jpeg");
+  background-size: cover;
```

Comment:
The background image here is broken. Fix it.

Reply:
ggghghgh